README.md

```diff
@@ -1,51 +1,78 @@
-# pgbouncer Foreign Data Wrapper
+# pgBouncer Foreign Data Wrapper
 
 ## Introduction
 
-pgbouncer_fdw provides a direct SQL interface to the pgbouncer SHOW commands. It takes advantage of the dblink_fdw feature to provide a more typical, table-like interface to the current status of your pgbouncer server(s). This makes it easier to set up monitoring or other services that require direct access to pgbouncer statistics.
+pgbouncer_fdw provides a direct SQL interface to the pgBouncer SHOW commands. It takes advantage of the dblink_fdw feature to provide a more typical, table-like interface to the current status of your pgBouncer server(s). This makes it easier to set up monitoring or other services that require direct access to pgBouncer statistics.
```

Comment:
The canonical spelling of PgBouncer is PgBouncer.  :-)

Reply:
So it is!